.github/workflows/causal_lm_cpp.yml

```diff
@@ -716,6 +716,13 @@ jobs:
           && timeout 120s ./build/samples/cpp/visual_language_chat/visual_language_chat ./miniCPM-V-2_6/ cat.jpg
           <<< $'What is on the image?\nWhat is special on the image?'
 
+      - name: Run python chat sample
```

Comment:
You need to update `cmake --build ./build/ --config Release --target visual_language_chat -j` line by adding `py_generate_pipeline` target